composer.json

```diff
@@ -28,7 +28,7 @@
 	},
 	"require": {
 		"php": ">=5.3.0",
-		"data-values/data-values": "~1.0|~0.1"
+		"data-values/data-values": "~2.0@dev|~1.0|~0.1"
```

Comment:
Not sure if this makes sense here. Must be changed to 2.0 before releasing this, obviously.